packages/devtools_app/test/shared/analytics_prompt_test.dart

```diff
@@ -1,338 +0,0 @@
-// Copyright 2020 The Chromium Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-
-import 'package:devtools_app/src/service/service_manager.dart';
```

Comment:
why was this test removed?

Reply:
Because the method under the test is not in use. So, I deleted both.

Reply:
We definitely use the AnalyticsPrompt in DevTools. If something is showing up as unused it is likely a false positive due to conditional imports